sdk/identity/identity/test/manual-integration/AzureFunctions/IdentityTest/.gitignore

```diff
@@ -0,0 +1,43 @@
+bin
+obj
+csx
+.vs
+edge
+Publish
```

Comment:
Let's try removing this file and see if anything is not properly ignored. I bet a lot of these are already ignored in our repo